### DISCUSSION TITLE
장소 등록 메서드 이름을 뭘로 할까요❓

### DISCUSSION THREAD

**Discussion by joong2043** [likelion-backendschool/your-anmok#46] [Q&A] [answered]
1. addPlace 
2. createPlace

더 좋은 이름 생각나면 말씀해주세요 !!

**Comment by himJJong** [comment]
저는 2번이요~

**Comment by Aruming** [comment]
저는 addPlace 에 한 표 던질게욥

**Comment by lion0913** [comment, marked as answer]
둘다 가독성이 뛰어난 것 같아서 뭘 해도 괜찮을 것 같지만 개인적으로는 addPlace 가 더 좋은것 같습니다!

장소를 우리가 생성하는게 아니라 기존에 있던 카카오 데이터를 기반으로 추가하는 개념인 것 같아서요!!

**Comment by ap3334** [comment]
저도 1번이요!
